.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Comment:
Minor nit: Can you change this to `true`. It keeps files from showing up in github's ui with a little icon saying it's an error. No biggie if you need it for some reason though!

Reply:
fixed